scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -931,8 +931,20 @@ wait_for_listen()
 
     for i in {1..300}
     do
-        ss -p state listening "( sport = :$PORT )" | grep -qE 'socat|nc' && break
         sleep 0.2
+        # List only our (mysql user) processes to avoid triggering SELinux
+        for cmd in $(ps -u $(id -u) -o pid,comm | sed 's/^\s*//g' | tr ' ' '|' | grep -E 'socat|nc')
```

Comment:
If you have multiple SST running, they can't all be on the same TCP port so the following grep on line 945 will sort out.  I realize some changes are required to account for different listening IP instead of 0.0.0.0 and likely also for IPv6.

Reply:
Yes, but the port check comes after the pid listing.  What happens is that we get a list of processes from the ps, some of those processes are then shut down since they are for a different SST, and we then get an error when trying to do the "ls -l /proc/$pid/fd" on a process that is no longer there.

Reply:
Would `| grep -E $(whoami) |` help also?

Reply:
This should be closed and refiled as a high priority jira bug instead.  PRs are meant for us to merge into the github tree directly (and we can't add commits to your branch to make changes).

Reply:
Ok, will do.